tests/unit/reactor_test.py

```diff
@@ -182,24 +183,29 @@ def test_wake(self):
         self.assertFalse(waker.awake)
         waker.wake()
         self.assertTrue(waker.awake)
+        # Wait until the reader becomes readable
+        select.select([waker._reader], [], [], 10)
```

Comment:
Would it make sense to move this to its own function, like: `waker.wait_reader()`?

Reply:
I need this just for tests, I don't want to add a function for such a single liner